<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDialog</class>
 <widget class="QDialog" name="CSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Preferences</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Difficulté</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="LevelComboBox">
        <item>
         <property name="text">
          <string>Ultra Débutant</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Débutant</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Difficile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ultra Difficile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chuck Norris</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="CustomCheckBox">
        <property name="text">
         <string>Paramétres Personnalisés</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Largeur</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="heightSpinBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hauteur</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="widthSpinBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bombes</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="sheepSpinBox"/>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
